woocommerce/sensei-certificates
# Add "View Certificate" button to Course Completed Actions block

Fixes #270.

### Changes proposed in this Pull Request

* Adds a _View Certificate_ block that is only available for pages.
* Adds the block to the _Course Completed_ block.
* Opens the certificate in the same tab when button is clicked.

### Testing instructions

* Use [this branch](https://github.com/Automattic/sensei/pull/4266) of Sensei.
* Ensure the _Course Completed Page_ setting has been configured.
* Add the _Course Completed Actions_ block to the _Course Completed_ page.
* Ensure the _View Certificate_ block appears.
* Add a certificate template to a course.
* Complete the course. On the _Course Completed_ page, click the _View Certificate_ button and ensure it opens the certificate in the same tab.
* Remove the `?course_id` parameter from the URL and ensure the button is no longer visible.
* Try adding a custom URL to the block in the editor. Ensure it's overwritten by the certificate's URL.
